### DISCUSSION TITLE
Can't sync ord. Stuck at 775k/777k due to:  error: JSON-RPC error: transport error: HTTP response too short: length 0, needed 12.

### DISCUSSION THREAD

**Discussion by jehphorey** [ordinals/ord#1800] [Q&A]
Hello, 

For some reason, I can't seem to completely sync ord. Once it reaches to max block (777311), I receive this error shown in the ss below. 
![image](https://user-images.githubusercontent.com/86658218/219939941-75414055-0bf8-438e-9bd7-3fb12f0ac9e8.png)

And when I use `ord index` again, it restarts back at 775k and restarts the process. Any ideas on what the issue could be? Only thing I can do right now is create an ord wallet with `ord wallet create` and receive btc because that doesn't require sync. 

Thank you. 

EDIT: Never mind, figured it out. It was simple. I should've put in a little more effort haha.

**Comment by benbodhi** [comment]
Would love to know how you fixed it! @jehphorey

**Reply by jehphorey** [reply]
Sure, do steps 6 through 15 before performing `ord index`

http://www.techlatest.net/support/bitcoin_fullnode_support/aws_gettingstartedguide/

**Reply by benbodhi** [reply]
Thanks for your reply.
I'm already connecting to my node on local network from another machine. Unfortunately think I may have a different issue.

**Reply by beleka** [reply]
Hi, step 6-15 is just to connect to your Ubuntu environment, I can't understand it. I'm facing the same issue and my bitcoin core node is perfectly synced and running in the background.

**Comment by ipensatori** [comment]
Similar problem here: 

- run "ord wallet inscribe", spend 3 days waiting for the indexing blocks to complete, 
- right at the end I get "JSON-RPC error: transport error: Couldn't connect to host: An established connection was aborted by the software in your host machine. (os error 10053)"

Upon rerunning the wallet inscribe command it restarts at block 775000, gets right to the end and then pops up the error above.

**Reply by belovie** [reply]
Did anything come up that helped you? Same issues here and I've done all I can think of/find.

**Comment by AryanJ-NYC** [comment]
> Hello,
> 
> For some reason, I can't seem to completely sync ord. Once it reaches to max block (777311), I receive this error shown in the ss below. ![image](https://user-images.githubusercontent.com/86658218/219939941-75414055-0bf8-438e-9bd7-3fb12f0ac9e8.png)
> 
> And when I use `ord index` again, it restarts back at 775k and restarts the process. Any ideas on what the issue could be? Only thing I can do right now is create an ord wallet with `ord wallet create` and receive btc because that doesn't require sync.
> 
> Thank you.
> 
> EDIT: Never mind, figured it out. It was simple. I should've put in a little more effort haha.

What a shite edit.

**Comment by bombless** [comment]
I tried
```
ord --bitcoin-data-dir /Volumes/My\ Passport wallet inscribe bombless.txt
```
ended up with this error message, and then I changed to
```
ord --bitcoin-data-dir '/Volumes/My Passport' wallet inscribe bombless.txt
```

it successed.

not sure if it's the case
